internal/config/config.go

```diff
@@ -186,6 +189,27 @@ func ResolveNVIDIACTKPath(logger logger.Interface, nvidiaCTKPath string) string
 	)
 }
 
+// ResolveNVIDIACDIHookPath resolves the path to the nvidia-cdi-hook binary.
+// This executable is used in hooks and needs to be an absolute path.
+// If the path is specified as an absolute path, it is used directly
+// without checking for existence of an executable at that path.
+func ResolveNVIDIACDIHookPath(logger logger.Interface, nvidiaCDIHookPath string) string {
```

Comment:
In responding to https://github.com/cncf-tags/container-device-interface/issues/203#issuecomment-2117628618 I had the thought that the concept of a CDI Hook executable could be standardized so that the `nvidia-cdi-hook` is a vendor-specific implementation. Does that mean that we should use `cdiHookPath` instead of using `nvidiaCDIHookPath` for our variables and config values?

Reply:
I don't think it matters all that much. Until such time as that actually happens, might as well get this in and iterate on it?